app/models/workflow_step.rb

```diff
@@ -2,7 +2,7 @@
 
 # Models a process that occurred for a digital object. Basically a log entry.
 class WorkflowStep < ApplicationRecord
-  validates :druid, presence: true
+  validates :druid, format: { with: /\Adruid:[a-zA-Z]{2}\d{3}[a-zA-Z]{2}\d{4}/, message: 'is invalid format or missing prefix' }
```

Comment:
the extra dependency might be overkill, but seemed worth mentioning druid-tools as an option to keep from more copy/paste of this regex.  https://github.com/sul-dlss/druid-tools/#check-whether-a-druid-is-valid

Reply:
yeah, that works...still need to check for the prefix separately though, alas:

```
DruidTools::Druid.valid?('bb123cd4567')
==> true
```

Reply:
ah, good catch, didn't realize that